### ISSUE TITLE
Cannot set aria-label for Toggle and ToggleSmall

### ISSUE THREAD

**Issue by alon24** [carbon-design-system/carbon#7196]
I cannot set aria-label for a toggl. When setting aria-label on Toggle or ToggelSmall,  it add the aria-label to the label and NOT to the input, so basically you have a label with aria-label, I think this is a bug
Toggle:
```
return (
      <div className={wrapperClasses}>
        <input
          {...other}
          {...checkedProps}
          aria-label={null}
          type="checkbox"
          id={id}
          className={`${prefix}--toggle-input`}
          onChange={(evt) => {
            onChange && onChange(evt);
            onToggle(input.checked, id, evt);
          }}
          ref={(el) => {
            input = el;
          }}
          onKeyUp={(evt) => {
            if (match(evt, keys.Enter)) {
              input.checked = !input.checked;
              onChange && onChange(evt);
              onToggle(input.checked, id, evt);
            }
          }}
        />
        <label
          className={`${prefix}--toggle-input__label`}
          htmlFor={id}
          aria-label={
            typeof labelText === 'string' ? null : this.props['aria-label']
          }>
          {labelText}
          <span className={`${prefix}--toggle__switch`}>
            <span className={`${prefix}--toggle__text--off`} aria-hidden="true">
              {labelA}
            </span>
            <span className={`${prefix}--toggle__text--on`} aria-hidden="true">
              {labelB}
            </span>
          </span>
        </label>
      </div>
```
And ToggleSmall:
```
return (
    <div className={wrapperClasses}>
      <input
        {...other}
        {...checkedProps}
        aria-label={null}
        type="checkbox"
        id={id}
        className={`${prefix}--toggle-input ${prefix}--toggle-input--small`}
        onChange={(evt) => {
          onChange && onChange(evt);
          onToggle(input.checked, id, evt);
        }}
        ref={(el) => {
          input = el;
        }}
        onKeyUp={(evt) => {
          if (match(evt, keys.Enter)) {
            input.checked = !input.checked;
            onChange && onChange(evt);
            onToggle(input.checked, id, evt);
          }
        }}
      />
      <label
        className={`${prefix}--toggle-input__label`}
        htmlFor={id}
        aria-label={ariaLabel}>
        {labelText}
        <span className={`${prefix}--toggle__switch`}>
          <svg
            className={`${prefix}--toggle__check`}
```
So the result is aria-label on the label 
```
        <Toggle
          id={`${id}-toggle`}
          name={`${namePrefix}Active`}
          className='toggle-with-time__toggle'
          toggled={pickerActive.value}
          labelA=''
          labelB=''
          disabled={disabled}
          aria-label={'Do you want your secret to expire?'}
        />
```
![image](https://user-images.githubusercontent.com/5415443/97882908-da1c8200-1d2c-11eb-8058-7760357aa5db.png)

**Comment by alon24** [comment]
actually JAWS sees the value correctly and only osx voice over does not see it correctly so I think this can be closed, because we support jaws
